xml/ay_bigfile.xml

```diff
@@ -2752,8 +2750,8 @@ openssl x509 -noout -fingerprint -sha256
   &lt;/drive&gt;
 &lt;/partitioning&gt;</screen>
      <para>
-      For a list of all possible &lt;skip_key&gt;, run <command>yast2
-      ayast_probe</command> on an already installed system.
+      For a list of all possible &lt;skip_key&gt;s, run <command>yast2
+      ayast_probe</command> on an already-installed system.
```

Comment:
Not sure about the hyphen...

Reply:
"An already installed system" is not very grammatic, but there are other possible fixes.

Reply:
Reworded to avoid the hyphen.